angular/base/ngsw-config.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
Can this file go in `src/`?

Reply:
No, as far as I can tell, it needs to be in the root dir